cluster/examples/kubernetes/ceph/upgrade-from-v1.4-crds.yaml

```diff
@@ -30,6 +30,10 @@ spec:
                   type: number
                 requireSafeReplicaSize:
                   type: boolean
+                replicasPerFailureDomain:
```

Comment:
I'd suggest we not worry about updating this file in this PR. We need to do it anyway with the v1 apiextensions conversion #6424.